src/client/dfs/dfs.c

```diff
@@ -985,6 +996,24 @@ open_sb(daos_handle_t coh, bool create, dfs_attr_t *attr, daos_handle_t *oh)
 	return rc;
 }
 
+int
+dfs_get_sb_layout(daos_key_t *dkey, daos_iod_t *iods[], int *akey_count,
+	 int *dfs_entry_size)
+{
+	if (dkey == NULL || akey_count == NULL)
+		return EINVAL;
+
+	D_ALLOC_ARRAY(*iods, SB_AKEYS);
```

Comment:
I think we need a routine to free these.

Reply:
Absolutely, `libdfs.free(iods)` was added